### ISSUE TITLE
[UI] [android] wrong yubikey shown when 5C plugged in

### ISSUE THREAD

**Issue by kzshantonu** [Yubico/yubioath-flutter#978]
Picture of 5C NFC shown when 5C plugged in

**Comment by AdamVe** [comment]
Hi, we are looking into this issue. Is it possible that you share output of `ykman --diagnose` for the key which shows the wrong image in the app? Let me know if you would need help.

**Comment by kzshantonu** [comment]
Sure. Also, the desktop app on Windows 10 does show the image correctly.

```

-----BEGIN PGP MESSAGE-----

hQIMA1gvg9ZVKE5/ARAAtL3uDtNbCLfgQ1PZmzqrqQ87y2ecbB+pBsfP2POwF3lv
q1qVirlY13Nzf+SBxNU7Pe0iXCk4CJDPG7xlZcrtroL2mK8cFhhLntpz4wRAj4RL
C/ucPDOoLp/q37j40SGfPV6UXNDNk/j93hUqMfk8Eh5wjDvCckP+R3eMn6XRHl2B
kbTj2zKCOt8Q2h9CPO+5QW2aW/7u5BtqiqflT9l3WLDlmvwowmxbNEMbrB6SKIFg
VkMT/rfs2B/zLmE6UEcouIsrJTr9m4Zkp/HrkUFtBuBk9EY3I0Ti35dcpC3G2hZi
ZxpOPr/eMTsGWe4+5XT0fTut/AL798LCI2TLWpv/NfKwzERZJ8ny4Kgv6He5PDBc
MmBasZ3Sq0lUjXBmD9+cOWGE5iOO5pbY2eNDAeHCumdNOzRJarLmtOg2/zfxoCiQ
aM3a/U8+y5jjr3SIz8kTtaVXWRweLCtK7sfR6/Sxqu2rd+NkHjKy6mvJlpzkv9pp
rbRMNXnOzbomnSs4Ri1gV1oJcKtjpKMTHtqWoQqOyZRmXDpoCxNeG+tcjnupa4gM
wlIMPaL5AHAgEkeAsADEYCOFt13gZ2iobodsNyowjeBaM2J3xZ59uaoxMn5OElJK
BmWL7JM72r0tk18poVQ7Q97yoUucN66qgxkgzevyflsEoLEdP1G0yuqlXUXmS8uF
AgwDHcke0a8u1BABEACE6svNVUoVOBzZ1qus3ty09UQq5GGLQMFQXksK5MmEsigW
0755ZS2qQGjWcspAk7uJuosv7Slv7QbpuHm8VGoS+q4qd1sTfRnyCnwvG25dwa7m
ynhq0sKueMqks1jkZkqFDwLxgyctDgTiL+JeFt8pFDKp/Hx33TV+4Qzx1/8KQl5C
bo8OGvfmBXSSfk0P6jj8CYUiePKcKtzqvOcITZz83nAWi8wv/mHds1PxOLjJqnNN
7U9LUb8HTSF02vmpov20AdHUD5UUJRGxbIgpg8qwtucKpDsMmLXT9nM3qWI3sIrH
YhDnXOcwd7XeGEb8Vpcc4EEnnrnMVUyMw+RRtzg55FlGvbgI5gq/VWforHYM2Amv
dlixQvGHHA1HUn/OFGTfWkUJwjqGSa/3BCIBR7jkAVgeSjioD6YWQI2gNjr6mvSz
l0KNck1rKqiDNVUidCnNGwId3d9qwjBH/En7KC5fufaxFfofDmJ+W5urWUsCiZVt
c4K1JFvQbq7N0tjWabjbVxA0vhx5U4k7Q3wqEeKwCa2tapr4pQIb2k9a+5C09yiH
EJJgj4gIPZCFkk45P4yIBBUCFBjTsXj/dlWwyqcN+5SJLgK3rVxvymsDz3dAmscl
x9iZRPMXPF5Zs54v3HEyi8yK+5pnwm4tUpBTfpRFIxQcVUwtY36/PgbIWQ948NLp
AZI/pgJnkSxZEMZbjTXd5bJlT1qoBV6eDq+rueQxroyIsx8+taXfUP++iwyzYzis
4UBaUoMvBB2bMZiiK8UE7+Y6YBukuNjHHwVNaNxqS8vF8WxttAKqS8yHao6Ctj94
uCJGykjNWCgDpdpFNXeZQPwTi5Opspc+7g5cwLNJUD1cjNo4nnfFEbbaqGFllRh/
q25IZH3aObOlnNnvg+ifJPQUeOUrjv+ZWvncBjREuQMNaZ5JmZGduHLUI26gCdSD
l5YHoqeM7VMtSxa/nSLphunfjuLFFExVIl4pXe5OkmWBwU6Sv1dJo2Z0gyuYLFti
9WLHqHQBXTUcmtbWLn2cQwEC5Kox31tvMpkcZJsv3tkcwej955J4y/G4YxCaZb/O
BJ2IiCIJrUAS38tyz2bUIRpTWqignP5MPx9W4Ukm1am3Be3N1wL03RfMibKL/FDX
zOaIKZMy6rpDsBH/ckYSjEK00mbwpZDYtijq6lb+AcVCu2stWiOJLRhuuBeNWczR
lfAu8FbcjYSyy3vgkflGGRsTm+L43I01/TOpVMoCMehORHYqgroTE5FjxC2r4IAX
Gd40wKx3x8Ql0uKHOibU7h6WVuKDGdcEEBcEWCyoBPzbpl6JEprv5zR5bjiVn+Ns
KOzDSLfTCTi08V2c+Hb7HqZZSSHw6des50cv+RK8mJPAs5O3imDaQsxTW8iyOh1F
bJeBwBrbLNWXr/FA+H2X1QQMW7NhCJFIbP5BR4L5XscfBKGq52/JMzURvkk25UEL
u2cGGz2hYAXP7yDPVKIh0l0vBtCCxVQEBr8sF4fjabZuRDNMrpF7u2n6lQoxe66Y
VQaTnv5saIVxdMrgM311Ao0Kg/q+U7I9pZMlq5p4Ki7TGJ4eMjiFmDvY2fwmlU8t
YxOlK4Q9+ETCE//9mQ2h5D9/ME2uV0pPg6LoTn0Xn7CCf4sXMaclHjR/V7BKpp5l
sRb4w+HbqCamHNp0BrNvYs5ctoBDHq3ROkidf7mnG2TbxGTeyoRV6uxfI4RY8RQG
XlqphTrlnyJdWsZZtRmjGRkD2iErGtbrbxctrTXZmKrQfxDRZyPXqMQqqZ21lDaM
SKejqT396+aMHZxoQJgam0FLOCNmSTO9ihNP2hOYtWIkEWAgx2TakLm4OloaS79p
SkjreF0Yfw+PUzrF+2I3M2TJR0SP
=Lg9e
-----END PGP MESSAGE-----


```

**Comment by AdamVe** [comment]
Thanks for the additional info.

**Comment by dainnilsson** [comment]
This is now fixed in 6.2.0.
